contracts/opsProxy/OpsProxy.sol

```diff
@@ -63,7 +63,15 @@ contract OpsProxy is Proxied, IOpsProxy {
         bytes calldata _data,
         uint256 _value
     ) private {
-        _call(_target, _data, _value, true, "OpsProxy.executeCall: ");
+        (, bytes memory returnData) = _call(
```

Comment:
Isn't that modifying the bytecode of the OpsProxy?

Reply:
Yea i am reverting the changes made in #123  which will keep OpsProxy untouched 😅